### PR TITLE
4 eyes functionality

### PR DESCRIPTION
**Pull request by matthewelwell**

### CODE REVIEW

File: api/tests/integration/features/workflows/test_integration_workflows.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    # when we update, let's just change the title
+    change_request_json = retrieve_response.json()
+    new_title = "%s (edit)" % change_request_json["title"]
+    change_request_json["title"] = new_title
+    update_response = admin_client.put(
+        cr_detail_url,
+        data=json.dumps(change_request_json),
+        content_type="application/json",
+    )
+    assert update_response.status_code == status.HTTP_200_OK
```
Comment (gagantrivedi) [inline api/tests/integration/features/workflows/test_integration_workflows.py:121]: Should we assert that `update_response.json()["title"] == new_title`?

File: api/users/models.py [hunk 2]
```diff
@@ -288,15 +288,17 @@ def send_alert_to_admin_users(subject, message):
 
     @classmethod
     def send_organisation_over_limit_alert(cls, organisation):
```
Comment (gagantrivedi) [inline api/users/models.py]: Is this related to this PR? Or it just slipped in?

File: api/tests/integration/features/workflows/test_integration_workflows.py (hunk 1 above)
Comment (matthewelwell) [reply api/tests/integration/features/workflows/test_integration_workflows.py:121]: We definitely should yes, I'll do that now.

File: api/users/models.py (hunk 2 above)
Comment (matthewelwell) [reply api/users/models.py]: Ah, yes, that just slipped in I think but I think it needs doing - I can't remember if I did it in another branch somewhere as well. Leave it with me...

File: api/tests/integration/features/workflows/test_integration_workflows.py (hunk 1 above)
Comment (matthewelwell) [reply api/tests/integration/features/workflows/test_integration_workflows.py:121]: Added.

File: api/users/models.py (hunk 2 above)
Comment (matthewelwell) [reply api/users/models.py]: I've removed this from here and created a separate PR here: #888.

**Review by gagantrivedi — APPROVED** [review]
